include/z64actor.h

```diff
@@ -38,11 +37,45 @@ typedef struct {
     /* 0x1C */ ActorFunc draw; // Draw function
 } ActorInit; // size = 0x20
 
-typedef enum {
-    /* 0 */ ALLOCTYPE_NORMAL,
-    /* 1 */ ALLOCTYPE_ABSOLUTE,
-    /* 2 */ ALLOCTYPE_PERMANENT
-} AllocType;
+/**
+ * @see ACTOR_ALLOC_ABSOLUTE
+ */
+#define ACTOR_OVERLAY_ABSOLUTE_SPACE_SIZE 0x27A0
+
+/**
+ * The actor overlay should be allocated memory for when loading,
+ * and the memory deallocated when there is no more actor using the overlay.
+ *
+ * `ACTOR_ALLOC_` defines indicate how an actor overlay should be loaded.
+ *
+ * @note Bitwise or-ing `ACTOR_ALLOC_` types is not meaningful.
+ * The `ACTOR_ALLOC_` types are 0, 1, 2 but checked against with a bitwise and.
+ *
+ * @see ACTOR_ALLOC_ABSOLUTE
+ * @see ACTOR_ALLOC_PERSISTENT
+ * @see actor_table.h
+ */
+#define ACTOR_ALLOC_NORMAL 0
+/**
+ * The actor overlay should be loaded to "absolute space".
+ *
+ * Absolute space is a fixed amount of memory allocated once.
+ * The overlay will still need to be loaded again if at some point there is no more actor using the overlay.
+ *
+ * @note Only one such overlay may be loaded at a time. This is not cleanly asserted.
```

Comment:
Not sure what you're going for with the last sentence, could you clarify?

Reply:
hes saying that only one overlay can be loaded to the absolute space at a time, and that its not clearly checked/asserted for this to be true. magic items can conflict and overwrite each other

Reply:
Is "This is not cleanly asserted" some sort of technical language I'm not familiar with? I'm not sure what it's supposed to mean if it isn't

Reply:
asserted in this context meaning checked/enforced. can be with an assert macro for example, or just checked in some capacity
its not clean in this case, cause it is indirectly enforced by most magic items restricting each other from executing while another is going. but its not as effective as it could be (as seen by some scenarios crashing due to an overlap in the absolute space)

Reply:
I think it should just say it in plain English rather than jargon, then. I had a look on Google and neither of those terms seems to be standardised (and "asserted" is ambiguous anyway, so seems even less appropriate). No reason to not be specific in documentation, it's not a variable name we want to keep short.